doc/mgr/ansible.rst

```diff
@@ -95,27 +116,62 @@ And use the "active" manager node: ( "ceph -s" command in one monitor give you t
 Operations
 ==========
 
+See the
+
 **Inventory:**
 
-Get the list of storage devices installed in all the cluster nodes. The output format is::
```

Comment:
Just wondering, if we instead should enhace the general orchestrator_cli documentation

Reply:
Agree.  Do you think that it could be interesting to add examples in each operation?
I also  have updated the list of operations implemented in the Ansible Orchestrator.

Reply:
having examples would be cool!

Reply:
jenkins retest this